### ISSUE TITLE
endless stack trace on master

### ISSUE THREAD

**Issue by freeformz** [heroku/barnes#7]
```console
$ heroku logs --tail
...
2017-10-19T19:01:53.567717+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `loop'
2017-10-19T19:01:53.567717+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `block in initialize'
2017-10-19T19:01:53.568670+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `/'
2017-10-19T19:01:53.568671+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `block in instrument!'
2017-10-19T19:01:53.568672+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `each'
2017-10-19T19:01:53.568672+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `instrument!'
2017-10-19T19:01:53.568673+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:23:in `block in instrument!'
2017-10-19T19:01:53.568673+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `each'
2017-10-19T19:01:53.568674+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `instrument!'
2017-10-19T19:01:53.568674+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:32:in `block (3 levels) in initialize'
2017-10-19T19:01:53.568675+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `each'
2017-10-19T19:01:53.568675+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `block (2 levels) in initialize'
2017-10-19T19:01:53.568675+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `loop'
2017-10-19T19:01:53.568676+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `block in initialize'
2017-10-19T19:01:53.569578+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `/'
2017-10-19T19:01:53.569579+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `block in instrument!'
2017-10-19T19:01:53.569580+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `each'
2017-10-19T19:01:53.569580+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `instrument!'
2017-10-19T19:01:53.569580+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:23:in `block in instrument!'
2017-10-19T19:01:53.569581+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `each'
2017-10-19T19:01:53.569581+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `instrument!'
2017-10-19T19:01:53.569581+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:32:in `block (3 levels) in initialize'
2017-10-19T19:01:53.569582+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `each'
2017-10-19T19:01:53.569583+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `block (2 levels) in initialize'
2017-10-19T19:01:53.569583+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `loop'
2017-10-19T19:01:53.569584+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `block in initialize'
2017-10-19T19:01:53.570490+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `/'
2017-10-19T19:01:53.570492+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `block in instrument!'
2017-10-19T19:01:53.570492+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `each'
2017-10-19T19:01:53.570493+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `instrument!'
2017-10-19T19:01:53.570493+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:23:in `block in instrument!'
2017-10-19T19:01:53.570494+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `each'
2017-10-19T19:01:53.570495+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `instrument!'
2017-10-19T19:01:53.570496+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:32:in `block (3 levels) in initialize'
2017-10-19T19:01:53.570496+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `each'
2017-10-19T19:01:53.570497+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `block (2 levels) in initialize'
2017-10-19T19:01:53.570497+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `loop'
2017-10-19T19:01:53.570497+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `block in initialize'
2017-10-19T19:01:53.571412+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `/'
2017-10-19T19:01:53.571413+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `block in instrument!'
2017-10-19T19:01:53.571414+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `each'
2017-10-19T19:01:53.571414+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `instrument!'
2017-10-19T19:01:53.571414+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:23:in `block in instrument!'
2017-10-19T19:01:53.571415+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `each'
2017-10-19T19:01:53.571415+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `instrument!'
2017-10-19T19:01:53.571416+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:32:in `block (3 levels) in initialize'
2017-10-19T19:01:53.571417+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `each'
2017-10-19T19:01:53.571417+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:31:in `block (2 levels) in initialize'
2017-10-19T19:01:53.571417+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `loop'
2017-10-19T19:01:53.571418+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/periodic.rb:20:in `block in initialize'
2017-10-19T19:01:53.572484+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `/'
2017-10-19T19:01:53.572487+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:74:in `block in instrument!'
2017-10-19T19:01:53.572487+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `each'
2017-10-19T19:01:53.572488+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/instruments/ruby_gc.rb:71:in `instrument!'
2017-10-19T19:01:53.572488+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:23:in `block in instrument!'
2017-10-19T19:01:53.572489+00:00 app[web.1]: /app/vendor/bundle/ruby/2.4.0/bundler/gems/barnes-61644edb837c/lib/barnes/panel.rb:22:in `each'
...
```

**Comment by freeformz** [comment]
Not sure this has anything to do with the feature flag actually.

**Comment by apg** [comment]
Not at all the feature flag. I committed the fix to master (08608404359a227283e80c94cc3003df461c830c). The railtie sets up 10 for the interval, and 60 for aggregation period, and because `10 / 60 = 0` in ruby, we ended up with a divide by 0 in ruby_gc.rb.

**Comment by freeformz** [comment]
Confirmed working.
